src/main/scala/io/digitallibrary/bookapi/service/translation/MergeService.scala

```diff
@@ -7,25 +7,33 @@
 
 package io.digitallibrary.bookapi.service.translation
 
+import java.io.ByteArrayOutputStream
+import java.net.URL
+import java.security.MessageDigest
+
+import org.apache.commons.codec.binary.Hex
 import com.typesafe.scalalogging.LazyLogging
+import io.digitallibrary.bookapi.BookApiProperties
+import io.digitallibrary.bookapi.integration.ImageApiClient
 import io.digitallibrary.bookapi.integration.crowdin.TranslatedChapter
 import io.digitallibrary.bookapi.model.api.Book
 import io.digitallibrary.bookapi.model.api.internal.NewTranslatedChapter
 import io.digitallibrary.bookapi.model.domain.{Chapter, Translation}
 import io.digitallibrary.bookapi.repository.ChapterRepository
+import javax.imageio.ImageIO
```

Comment:
burde denne ligge oppe med de andre java importene?

Reply:
Also vurderte du å se på den url sjekken som Jonas foreslo?

Reply:
Ja kan flytte den.
Vurderte det, men siden det bare er for admins så tenkte jeg at det ikke var nødvendig